Assets/Tests/Play/DepthkitPlayTest.cs

```diff
@@ -0,0 +1,149 @@
+    {
+        player.OnVideoStart += EventTriggered;
+        
+        player.SignIn(VALID_STREAMING_TOKEN);
+        player.PlayVideo(VALID_VIMEO_VOLUMETRIC_VIDEO_ID);
+
+        yield return new WaitUntil(()=> triggered);
+        Assert.IsFalse(String.IsNullOrEmpty(depthkitObj.GetComponent<VideoPlayer>().url));
+    }
+
+#if VIMEO_AVPRO_VIDEO_SUPPORT
+    [UnityTest]
+    public IEnumerator Can_Playback_Volumetric_Video_With_Valid_Token_And_AVPro_Video() 
+    {
+        depthkitClip.Setup(Depthkit.AvailablePlayerType.AVProVideo, Depthkit.RenderType.Photo, new TextAsset());
+        player.depthKitClip = depthkitClip;
+
+        //Set these so AVPro doesn't log errors about not having a file path
+        depthkitClip.GetComponent<RenderHeads.Media.AVProVideo.MediaPlayer>().m_AutoOpen = false;
+        depthkitClip.GetComponent<RenderHeads.Media.AVProVideo.MediaPlayer>().m_AutoStart = false;
+        
+        depthkitClip._needToResetPlayerType = true;
+
+        player.OnVideoStart += EventTriggered;
+
+        player.SignIn(VALID_STREAMING_TOKEN);
+        player.PlayVideo(VALID_VIMEO_VOLUMETRIC_VIDEO_ID);
+
+        yield return new WaitUntil(()=> triggered);
+        Assert.IsFalse(String.IsNullOrEmpty(depthkitClip.GetComponent<RenderHeads.Media.AVProVideo.MediaPlayer>().m_VideoPath));
+    }
+
+    [UnityTest]
+    public IEnumerator Can_Playback_Adaptive_Volumetric_Video_With_Valid_Token_And_AVPro_Video() 
+    {
+        depthkitClip.Setup(Depthkit.AvailablePlayerType.AVProVideo, Depthkit.RenderType.Photo, new TextAsset());
+        player.depthKitClip = depthkitClip;
+
+        player.selectedResolution = StreamingResolution.Adaptive;
+
+        //Set these so AVPro doesn't log errors about not having a file path
+        depthkitClip.GetComponent<RenderHeads.Media.AVProVideo.MediaPlayer>().m_AutoOpen = false;
+        depthkitClip.GetComponent<RenderHeads.Media.AVProVideo.MediaPlayer>().m_AutoStart = false;
+        
+        depthkitClip._needToResetPlayerType = true;
+
+        player.OnVideoStart += EventTriggered;
+
+        player.SignIn(VALID_STREAMING_TOKEN);
+        player.PlayVideo(VALID_VIMEO_VOLUMETRIC_VIDEO_ID);
+        
+        yield return new WaitUntil(()=> triggered);
+        yield return new WaitForSeconds(1.0f);
+        Assert.IsFalse(String.IsNullOrEmpty(depthkitClip.GetComponent<RenderHeads.Media.AVProVideo.MediaPlayer>().m_VideoPath));
+    }
+#endif
+
+    [UnityTest]
+    public IEnumerator Vimeo_Player_Sets_Volumetric_Metadata() 
+    {    
+        player.OnVideoStart += EventTriggered;
+        
+        player.SignIn(VALID_STREAMING_TOKEN);
+        player.PlayVideo(VALID_VIMEO_VOLUMETRIC_VIDEO_ID);
+
+        yield return new WaitUntil(()=> triggered);
+        Assert.IsNotNull(depthkitClip._metaDataFile);
+    }
+
+    [UnityTest]
+    public IEnumerator Vimeo_Player_Gracefully_Errors_When_Metadata_Is_Not_Valid()
```

Comment:
We can remove this test since we are testing these errors in the `VimeoVideo` unit tests. This is just duplicating errors in the output.